README.md

```diff
@@ -765,6 +766,105 @@ NB: Using the JSON Messsage Generator option it is possible to format messages d
 
   More info and an example can be found [here](https://aws.amazon.com/blogs/compute/continuous-integration-deployment-for-aws-lambda-functions-with-jenkins-and-grunt-part-1/)
 
+### Trigger a Lambda function when an email is received by Amazon Simple Email Service
+
+Start by creating a blank AWS Lambda function that will be called whenever a new email comes in.
+
+#### Set up SES
+For this you need your own domain, and you need to verify the domain with AWS, this can take up to 72hrs, so have a nice drink and chill out while you wait :sunglasses: :coffee:
+
+	See here for how: http://docs.aws.amazon.com/ses/latest/DeveloperGuide/verify-domain-procedure.html
+
+#### Add a rule set
+Click `rule sets` on the bottom left and create a new rule set with whatever name you like. If you want to have your function triggered by emails sent to any address in your domain you can leave recipient blank, else you can add the recipient you want to trigger the function.
+
+In that rule set add an action for your Lambda function. This tells AWS to run your action when an email is received by SES.
+
+This is all you need to do to make your function trigger from SES, but depending on what your Lambda function does, it might be kind of hard to verify that it's run (though you can look at the stats for the function to see how and when it's been run) so let's make it save the body of your email to your bucket!
+
+#### Save the email to S3
+Go back to the rule set you created for your Lambda function. Add a new action to it, this one should be an S3 action, with the bucket you want to use selected. This will save the email to S3. Make sure this action is positioned **above** your Lambda function:
+
+![ses management console - google chrome_006](https://user-images.githubusercontent.com/22300773/28177094-a0e16bfc-67f1-11e7-8676-feabc437295f.png)
```

Comment:
try to avoid having other tabs in the screenshot as it's "_noisy_".
(_keep the focus on the task you are currently performing to avoid distracting/confusing your reader_) 😉

Reply:
I don't have access to Photoshop here and the Linux alternatives are a bit of a pain to use. Happy to update this when I get home from my holiday :tropical_drink: